app/models/group.rb

```diff
@@ -424,6 +424,10 @@ def organisation_motions_count
     Discussion.published.where(group_id: org_group_ids).sum(:motions_count)
   end
 
+  def closed_motions_count
```

Comment:
Can't do this, unfortunately, it'll be too slow. If we want this feature, we'll have to make a counter cache column for it, `closed_motions_count`, on the group model.